### PR TITLE
63 booking widget changes

### LINKED ISSUE #63: The airport code sorting logic used is old one.

### PR DESCRIPTION
**Pull request by sselvara**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #63 

**_1. Updated JS/CSS and added the mock files to adhere the booking widget changes.
2. Uploaded new icons for calendar 
3. Added a single search mock API to adhere to the new implementation of booking widget that shows price under each date upon source/destination selection, it will work only for DEL to BLR in EDS domains. It should work for any combination in the subdomain as it will consume real API endpoint._**

Test URLs:
- Before: https://main--airindia--hlxsites.hlx.live/in/en/book/special-offers/global-connectivity
- After: https://63-booking-widget-changes--airindia--hlxsites.hlx.page/in/en/book/special-offers/global-connectivity

### CODE REVIEW

File: content/dam/air-india/currencyjson/currencyCode.json [hunk 1]
```diff
@@ -0,0 +1,612 @@
+[
```
Comment (sdmcraft) [inline content/dam/air-india/currencyjson/currencyCode.json:1]: Can we keep this info in excel? Also, is it necessary to keep it in `/content/dam/air-india`

**Review by sdmcraft — COMMENTED** [review]
All icons should ideally go in `icons` folder in Sharepoint or Git repo.

**Review by sdmcraft — COMMENTED** [review]
Also unrelated to this review but can we get rid of the liniting errors?

**Comment by sselvara** [comment]
Closing this PR as all these changes are also part of #70.

**Pull request closed without merging**